src/app/shared/utils/fields.utils.ts

```diff
@@ -122,4 +122,24 @@ export class FieldsUtils {
     return Object.assign({}, obj);
   }
 
+  showGrayBar(caseField: CaseField, element: HTMLElement) {
+    if (caseField && caseField.field_type && caseField.field_type.type !== 'Collection') {
```

Comment:
caseField check is not needed, remove please. In the code we are always passing a value for case field. And if that didnt' happen, the conditional show directive will break even before this method is called. 

Also caseField.field_type check is not required because fieldType is mandatory. If this breaks the tests, then the test have to be updated rather that polluting the production code with unnecessary null checks all over the place

Reply:
Done.  Removed those checks.   I had added them because it was failing several unit tests.  Now I fixed all the test cases.  All fine now.